common/src/test/scala/uk/ac/wellcome/test/fixtures/S3.scala

```diff
@@ -0,0 +1,69 @@
+package uk.ac.wellcome.test.fixtures
+
+import com.amazonaws.auth.{AWSStaticCredentialsProvider, BasicAWSCredentials}
+import com.amazonaws.client.builder.AwsClientBuilder.EndpointConfiguration
+import com.amazonaws.services.s3.model.S3ObjectSummary
+import com.amazonaws.services.s3.{AmazonS3, AmazonS3ClientBuilder}
+import io.circe.Json
+import io.circe.parser.parse
+import org.apache.commons.io.IOUtils
+import org.scalatest.Matchers
+import org.scalatest.concurrent.Eventually
+import uk.ac.wellcome.test.utils.ExtendedPatience
+
+import scala.collection.JavaConversions._
+import scala.util.Random
+
+trait S3 extends Eventually with ExtendedPatience with Matchers {
```

Comment:
I don't think `Eventually` is actually used in this trait?

Reply:
👍 leftovers from a revision where i thought i needed to check the bucket had been created